Dockerfile

```diff
@@ -14,21 +14,7 @@ RUN pip3 install --upgrade pip && pip install --upgrade wheel && \
 RUN mkdir -p /home/scaler/ /var/lib/f8a-scaler/
 
 COPY scale.sh sqs_status.py liveness_check.sh /home/scaler/
-COPY scalerjob /etc/cron.d/scalerjob
 
 WORKDIR /home/scaler
 
-RUN chmod 0644 /etc/cron.d/scalerjob && \
-    chmod 0777 /var/lib/f8a-scaler/ && \
-    chmod 0777 /var/run/
-
-# Workaround centos crond problem
-RUN sed -i '/session required pam_loginuid.so/d' /etc/pam.d/crond
-
-RUN touch /var/log/cron.log && \
-    echo 1 > /var/lib/f8a-scaler/liveness && \
-    echo 0 > /var/lib/f8a-scaler/liveness_prev
```

Comment:
Not 100% sure about this, but it's possible that liveness probe will fail it the file doesn't exist.

Reply:
Maybe I missed something but it should be created on first iteration - if not the liveness should fail (?).